src/main/kotlin/no/nav/familie/ef/sak/opplysninger/søknad/mapper/AktivitetMapper.kt

```diff
@@ -21,21 +21,21 @@ import no.nav.familie.ef.sak.vilkår.dto.VirksomhetDto
 
 object AktivitetMapper {
 
-    fun tilDto(aktivitet: Aktivitet, situasjon: Situasjon, søknadBarn: Set<SøknadBarn>): AktivitetDto {
+    fun tilDto(aktivitet: Aktivitet?, situasjon: Situasjon?, søknadBarn: Set<SøknadBarn>): AktivitetDto {
```

Comment:
Hvis vi betaler for generaliseringen noe sted er det kanskje ?-tegnene her. Men helt greit :)